src/Files.App/Data/Contracts/INetworkService.cs

```diff
@@ -3,24 +3,41 @@
 
 namespace Files.App.Data.Contracts
 {
-	public interface INetworkDrivesService
+	public interface INetworkService
```

Comment:
I was also thinking INetworkLocationsService, but since network computers are not included in network locations, I went with the more generic INetworkService.
```
Network --- Network computers
         |
         -- Network locations --- Network drives
                               |
                               -- Network shortcuts
```

Reply:
If you have rationale, it's fine 👍